src/en/administration/backup_and_restore.md

```diff
@@ -34,6 +34,13 @@ docker-compose up -d postgres
 
 # Restore from the .sql.gz backup
 gunzip < db_dump.sql  |  docker-compose exec -T postgres psql -U lemmy
+
+# Note: You may need to change the permissions on the postgres directory
+chown -R $USER volumes
```

Comment:
What is $USER?

Reply:
Almost always the current user in linux. Its just a note that they may need to do that. On my arch machine, starting up postgres by itself uses root, but starting it from docker-compose uses my user.

Reply:
I'll add a "depending on your system". Not much else I can say there.